test/pageobjects/page.js

```diff
@@ -14,7 +14,9 @@ const waitFor = function(expected) {
 };
 
 const goTo = function(uri) {
-  return browser.get(uri).then(() =>  waitFor(uri));
+  return browser.get(uri).then(() =>  waitFor(uri)).catch(() => {
+    console.error(`Failed to navigate to ${browser.getCurrentUrl()}`);
```

Comment:
Would be good to align on a way how we will display errors. Cause ATM we are displaying errors also as [console.log('[ERROR]' + error)](https://github.com/benjaminapetersen/origin-web-console-smoke-test/blob/master/test/server.js#L63)
Dont have any preferences but maybe it would be better to have the INFO / ERROR / WARNING prefix to mast bash.
Thoughts ? :)

Reply:
Yup! I should have used `logger`:

logger.error(`Failed to navigate to ${browser.getCurrentUrl()}`);

I believe I put the prefixes in the logger package so we don't have to manually do that:

```
// test/helpers/logger.js

  {key: 'log',      prefix: 'INFO',     color: COLORS.reset },
  {key: 'info',     prefix: 'INFO',     color: COLORS.blue },
  {key: 'warn',     prefix: 'WARN',     color: COLORS.yellow },
  {key: 'error',    prefix: 'ERROR',    color: COLORS.red },
  {key: 'success',  prefix: 'SUCCESS',  color: COLORS.green },

```

`logger` is async, so it collects the logs as promises & executes them in order after previous completes, so it should work here.  The raw `console.log` that we have occasionally are probably because the async part didn't work.  😦  

I should probably fix that so we just always use `logger` and let the prefixes be standardized.  Lemme see if I can do that.